### DISCUSSION TITLE
How to create a filter for multiple unrelated pages?

### DISCUSSION THREAD

**Discussion by gravelld** [plausible/analytics#2717] [Anything Else]
I've got a bunch of pages that I want to exclude from analytics. I've read https://plausible.io/docs/filters-segments#exclude-specific-segments-of-traffic but this appears to work on the basis of one value. What if I have multiple?

For example, my equivalent segment in GA was:

```
Not landing on pages (some-path/A OR another-path/B OR C)
AND
Device is not mobile
AND
Country is (US OR UK OR Western Europe)
```

I think the ANDs are covered by Plausible, but how to specify multiple unrelated values? E.g. there's no hierarchy or naming commonality with regards the landing pages. And I don't see any reference to having multiple equals or contains clauses.

Sorry if my searching was weak.

**Comment by metmarkosaric** [comment]
hi @gravelld! grouping unrelated pages is not possible at the moment unfortunately. we have a thread for that feature request here https://github.com/plausible/analytics/discussions/1528

**Reply by gravelld** [reply]
Thanks, I added a vote.

**Comment by metmarkosaric** [comment]
**Update**: You can now filter your stats by multiple countries, pages, sources and more, all at once. Click on the "Filter" button in your dashboard to start exploring.

**Reply by gravelld** [reply]
Just want to say thank you for this, it works really nicely.

It would be good to be able to name a filter set and use that in the URL, not sure if the query string is going to get too large eventually!

**Reply by metmarkosaric** [reply]
glad to hear that @gravelld! would be nice to introduce some type of "save filter" functionality in the future. for now bookmarking a link would work as the url changes with every filter change
